### ISSUE TITLE
Destination repository

### ISSUE THREAD

**Issue by arvkonstantin** [azolotko/sbt-gitlab#91]
Hello! In the original repository there is a configuration to which repository to publish, is it possible to do this using your plugin?

```
GitlabPlugin.autoImport.gitlabGroupId     :=  Some(12345)
GitlabPlugin.autoImport.gitlabProjectId   :=  Some(12345)
```

**Comment by azolotko** [comment]
Hi @arvkonstantin!

Yes, here's how [publishTo is defined](https://github.com/azolotko/sbt-gitlab/blob/75a3cbb5be62411147840df5d868d0131d259d53/src/main/scala/nl/zolotko/sbt/gitlab/GitlabPlugin.scala#L121C1-L126C9) at the moment:

```scala
publishTo := (ThisProject / publishTo).value.orElse(
  gitlabProjectId
    .map(
      GitlabProjectRepository(gitlabDomain.value, _).resolver
    )
)
```

**Comment by arvkonstantin** [comment]
@azolotko Thank you!

Can you add an example to the readme?

**Comment by azolotko** [comment]
@arvkonstantin I've updated the [README.md](https://github.com/azolotko/sbt-gitlab#publishing-outside-gitlab-cicd), but also discovered and [fixed a bug](https://github.com/azolotko/sbt-gitlab/pull/92/files#diff-80754f316ad38ed00df650ed2ff9b81823068b6cb56bc64fb456fba286c1e285R119) that prevented it from working.

A version with the fix (v0.1.4) should appear on Maven Central in couple hours. Alternatively, you can already get from `s01.oss.sonatype.org`, if you have it configured.

Please give it a try and let me know if there are any issues.

Thanks!
